runtime/src/accounts_db.rs

```diff
@@ -2103,4 +2132,56 @@ pub mod tests {
 
         Ok(())
     }
+
+    #[test]
+    fn test_hash_stored_account() {
+        // This test uses some UNSAFE trick to detect most of account's field
+        // addition and deletion without changing the hash code
+
+        const ACCOUNT_DATA_LEN: usize = 3;
+        // the type of InputTuple elements must not contain references;
+        // they should be simple scalars or data blobs
+        type InputTuple = (
+            Slot,
+            StoredMeta,
+            AccountMeta,
+            [u8; ACCOUNT_DATA_LEN],
+            usize, // for StoredAccount::offset
```

Comment:
I wanted to write this like `typeof(StoredAccount::offset)` but it seems I can't...